src/CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Contribute to Flutter docs
+description: Types of contributors and how to contribute to the Flutter project.
+---
+
+If you would like to help contribute to the Flutter project,
+we’re happy to have your help! Anyone can contribute, whether you’re new to
+the project or you’ve been around a long time, and whether you self-identify
+as a developer, an end user, or someone who just can’t stand seeing typos.
+
+For more ways to get involved in the Flutter community or to learn about us,
+also visit the [Flutter community](/community) page.
+
+Looking for the [Effective Dart]({{site.www}}/guides/language/effective-dart)?
+This comprehensive set of guides for the Dart language (which is used by Flutter),
+includes guides on style, documentation, usage, and design.
+
+* [Types of contributor](#types-of-contributor)
+* [Ways to contribute](#ways-to-contribute)
+
+# Types of contributor
+
+You can contribute to the Flutter SDK or the Flutter website. There are two
+levels of contributors:
+
+* A _contributor_ has signed the Contributor License Agreement (CLA) and
+  submittecd one or more patches.
+
+* A _trusted contributor_ is granted commit access by gaining our trust and
+  demonstrating a commitment to Flutter.
+
+# Ways to contribute
+
+We encourage you to reach out and join the conversation in a variety of ways.
+
+An easy way to send feedback is to "thumbs up" issues important to you
+in either the issue tracker for the [Flutter SDK and API docs][issues],
+or the [flutter.io website][doc-issues].
+
+Other ways you can contribute:
+
+* [Ask HOWTO questions that can be answered with specific solutions][so]
+* [Live chat with Flutter engineers and users][gitter]
+* [Discuss Flutter, best practices, app design, and more on our mailing list][mailinglist]
+* [Report bugs and request features][issues]
+* [Report API docs bugs][issues]
+* [Request docs for flutter.io][doc-issues]
+* [Follow us on Twitter: @flutterio](https://twitter.com/flutterio/)
+* [Read the Flutter Publication on Medium](https://medium.com/flutter-io)
+* [Sign up to Future UX Studies on Flutter](/research-signup)
+* [Join the subreddit to keep up with the latest in the Flutter community][reddit]
```

Comment:
we should probably also link to the contributing doc, for those who want to actually write code